modules/ob-understanding-the-status-of-a-build-run.adoc

```diff
@@ -0,0 +1,75 @@
+
+| `False` | `TaskRunGenerationFailed` | The generation of a `TaskRun` specification has failed.
+
+| `False` | `MissingParameterValues` | You have not provided any value for some parameters that are defined in the build strategy without any default. You must provide the values for those parameters in the `Build` or the `BuildRun` CR.
+
+| `False` | `RestrictedParametersInUse` | A value for a system parameter was provided, which is not allowed.
+
+| `False` | `UndefinedParameter` | A value for a parameter was provided that is not defined in the build strategy.
+
+| `False` | `WrongParameterValueType` | A value was provided for a build strategy parameter with the wrong type. For example, the parameter is defined as an array or a string in the build strategy and you must provide a set of values or a direct value accordingly.
+
+| `False` | `InconsistentParameterValues` | A value for a parameter contained more than one of these values: `value`, `configMapValue`, and `secretValue`. You must provide only one of the mentioned values to maintain consistency.
+
+| `False` | `EmptyArrayItemParameterValues` | An item inside the values of an array parameter contained none of these values: `value`,`configMapValue`, and `secretValue`. You must provide only one of the mentioned values as null array items are not allowed.
+
+| `False` | `IncompleteConfigMapValueParameterValues` | A value for a parameter contained a `configMapValue` value where the `name` or the `value` field was empty. You must specify the empty field to point to an existing config map key in your namespace.
+
+| `False` | `IncompleteSecretValueParameterValues` | A value for a parameter contained a `secretValue` value where the `name` or the `value` field was empty. You must specify the empty field to point to an existing secret key in your namespace.
+
+| `False` | `ServiceAccountNotFound` | The referenced service account was not found in the cluster.
+
+| `False` | `BuildRegistrationFailed` | The referenced build in the `BuildRun` resource is in a `Failed` state.
+
+| `False` | `BuildNotFound` | The referenced build in the `BuildRun` resource was not found.
+
+| `False` | `BuildRunCanceled` | The `BuildRun` and related `TaskRun` resources were canceled successfully.
+
+| `False` | `BuildRunNameInvalid` | The defined build run name in the `metadata.name` field is invalid. You must provide a valid label value for the build run name in your `BuildRun` CR.
+
+| `False` | `BuildRunNoRefOrSpec` | The `BuildRun` resource does not have either the `BuildRef` or `BuildSpec` field defined. 
+
+| `False` | `BuildRunAmbiguousBuild` | The defined `BuildRun` resource uses both the `BuildRef` and `BuildSpec` fields. Only one of the parameters is allowed at a time.
+
+| `False` | `BuildRunBuildFieldOverrideForbidden` | The defined `BuildRun` resource uses an override in combination with the `BuildSpec` field, which is not allowed. Use the `BuildSpec` field to directly specify the respective value.
```

Comment:
```suggestion
| `False` | `BuildRunBuildFieldOverrideForbidden` | The defined `spec.build.name` resource uses an override in combination with the `spec.build.spec` field, which is not allowed. Use the `spec.build.spec` field to directly specify the respective value.
```

Reply:
@apoorvajagtap Do we need to replace `The defined `BuildRun` resource` with `The defined `spec.build.name` resource`? Correct??

Reply:
Yes.

Reply:
@shipsing now that I read `resource` the following would be more appropriate:
~~~
The defined `spec.build.name` field uses an override in combination with the `spec.build.spec` field, which is not allowed. Use the `spec.build.spec` field to directly specify the respective value.
~~~